frontend/src/components/TimeEntry/EditEntryForm.js

```diff
@@ -61,19 +59,19 @@ const updatedValues = (timeUnit, allValues, start) => {
     case 'DAY': {
```

Comment:
same code AddEntryForm ?

Reply:
this could be part of a larger refacto but for me it's out of the scope of this PR.